static/app/utils/replayCount/useReplayCountForTransactions.tsx

```diff
@@ -13,15 +15,16 @@ interface Props {
  */
 export default function useReplayCountForTransactions({
   bufferLimit = 25,
-  statsPeriod = '14d',
+  statsPeriod,
 }: Props = {}) {
   const organization = useOrganization();
+  const location = useLocation();
   const {getOne, getMany, hasOne, hasMany} = useReplayCount({
     bufferLimit,
     dataSource: 'discover',
     fieldName: 'transaction',
     organization,
-    statsPeriod,
+    statsPeriod: statsPeriod ?? decodeScalar(location.query.statsPeriod) ?? '90d',
```

Comment:
The callsite is also doing `decodeScalar(location.query.statsPeriod)` so why does it need to be inside here as a default/fallback as well? i think we should keep this method simpler and straight forward like before.

Reply:
good point, i had this here before i realized the callsites weren't passing in `statsPeriod` at all. can remove this now